pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -172,10 +170,19 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		if err != nil {
 			return err
 		}
+
```

Comment:
The usage string (line [61](https://github.com/GoogleCloudPlatform/kubernetes/pull/7279/files#diff-57b5708f59a1279bfe3c9e55adaef5a1R61)) needs to be updated for the new syntax options.